docs/nuget-org/policies/deleting-packages.md

```diff
@@ -34,3 +34,25 @@ Packages that meet any of the following criteria are not allowed on the public N
 If you find a package that is in violation of any of these items, click the **Report Abuse** link on the package details page and submit a report.
 
 Note that the NuGet team and the .NET Foundation reserves the right to change these criteria at any time.
+
+## Unlisting a package
+The purpose of unlisting a package version is that it hides the package version from search and from nuget.org package details page. Furthermore it allows existing users of the package to keep downloading it, but reduces new adoption since the package is not visible in search.
+
+In order to unlist a specific package version please follow these steps:
+
+** Unlist a specific package version **
+- Click the `Account name` at the top right corner. 
+- Click `Manage packages`
+- Click `Published packages`
+- Click the package name which version you want to unlist
+You will now see all the versions on that package. 
+- Under the `Status` column click the `Listed` link on that package version you want to unlist.
+- You will now be transferred to the "Manage" page of that specific nuget package version. 
+- Click the plus sign beside the "Listing" and uncheck the checkbox that says: “List in search results”.
+- Click the “Save” button.
+
+** Verification **
+The specific package version has now been unlisted. In order to verify this open a incognito instance of your browser and move to the URL of the package (without the version part) e.g.: https://www.nuget.org/packages/YOUR-PACKAGE-NAME/. You will see all versions of that package that have * * not * * been unlisted. However if you see the same page while logged in you will see all packages with their status; listed or unlisted. 
+
+It's also possible to deprecate a package version (incase you can't delete a package version). For more information about deprecating package versions, see [Deprecating packages](Deprecate-packages.md).
+
```

Comment:
```suggestion
## Unlisting a package
Unlisting a package version hides it from search and from nuget.org package details page. This allows existing users of the package to continue using it but reduces new adoption since the package is not visible in search.

Steps to unlist a package:

1. Select `Your account name` (at the top right corner) >`Manage packages` > `Published packages`
1. Select the "Manage package" icon
1. Expand the "Listing" section and select the package version
1. Uncheck “List in search results” and select "Save"

The specific package version has now been unlisted. In order to verify this open an incognito instance of your browser and navigate to the package page (without the version part) e.g.: https://www.nuget.org/packages/YOUR-PACKAGE-NAME/. You will see all versions of that package that have **not** been unlisted. However, if you see the same page while logged in you will see all packages with their status; listed or unlisted. 

It's also possible to deprecate a package version (in case you can't delete a package version). For more information about deprecating package versions, see [Deprecating packages](Deprecate-packages.md).

```